ui/src/Llm.tsx

```diff
@@ -27,6 +28,37 @@ export const rephraseQuestion = async (question: string, conversation: any[], mo
     }
 }
 
+export const get_filters_prompt = (text: string) => {
+    const sys = "Given the following question, extract the value of the following attributes if available. If you cannot confidently extract the value say unknown.\n" +
+        "1. location\n \
+    2. airplane_name\n \
+    3. date_start in yyyy - mm - dd\n \
+    4. date_end in yyyy - mm - dd\n \
+    Return response as a json\
+"
+    const prompt = text
+    const openAiPrompt = [{ role: "system", content: sys }, { role: "user", content: prompt }]
+    return openAiPrompt;
+}
+export const getFilters = async (question: string, modelName: string) => {
+    const prompt = get_filters_prompt(question)
+    const chatJson = JSON.stringify({
+        stream: false,
+        model: "gpt-3.5-turbo-1106",
```

Comment:
modelName appears not to be used. plug into model field?

Reply:
Done